src/solver/type_getters.jl

```diff
@@ -958,8 +958,14 @@ function get_simulation(config::AtmosConfig)
                     )
                 else
                     # Add to the accumulator
+
+                    # We use similar + .= instead of copy because of a bug in CUDA.jl 5.2.0
```

Comment:
Can you please provide a link to the issue?

Reply:
There's no issue yet, I asked on slack #gpu, if someone confirms that this is a problem, I will open the issue.

Reply:
I was pointed out to this https://github.com/JuliaGPU/Adapt.jl/issues/21 with the following comments:
> because of how Julia works, it's tricky to define GPU-accelerated functions for nested array wrappers, so this is triggering fallback functionality from Base
> this would need a change in Base, or a more radical type system change, to make this easier. alternatively, relying more on unified memory would make the fallbacks work better, which is probably what we're going to do instead.